src/api/demo.rkt

```diff
@@ -18,25 +18,18 @@
          "../config.rkt"
          "../syntax/read.rkt"
          "../utils/errors.rkt")
-(require "../syntax/types.rkt"
-         "../syntax/sugar.rkt"
-         "../utils/alternative.rkt"
+(require "../syntax/sugar.rkt"
          "../core/points.rkt"
-         "../api/sandbox.rkt"
-         "../utils/float.rkt")
+         "../api/sandbox.rkt")
 (require "datafile.rkt"
          "../reports/pages.rkt"
          "../reports/common.rkt"
          "../reports/core2mathjs.rkt"
-         "../reports/history.rkt"
-         "../reports/plot.rkt"
          "server.rkt")
```

Comment:
Deleting unused stuff is good but I'd rather anything dealing with HTML or HTTP live in `demo.rkt` instead of `server.rkt`. The server should focus on start/stopping/status of jobs, and caching. Basically "routing". Ideally actual "view" stuff, like what pages to generate, should live elsewhere, as should details of how we send HTTP responses.